### PR TITLE
Proper Documentation and project cleanup

### LINKED ISSUE #2: Add Proper Docs
Add docs on: README, CONTRIBUTION

### LINKED ISSUE #3: Add Example
Add example file how to use this module

### PR DESCRIPTION
**Pull request by k1m0ch1**
# Description

1. change the `README.md` with the minimum documentation to run the program
2. add `package.json` so everyone can easily to run the program
3. change `index.js` to add the input prompt in case user need to do this manually and the instruction to turn this on/off on the documentation

Fixes #2 

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)
- [X] This change requires a documentation update

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have checked my code and corrected any misspellings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,58 @@
...
+# How to Run Program
+
+At first you need to specify this three variable `Cookie`, `Latitude` and `Longitude`
+
+## How to get Cookie?
+
+1. you need to login with this url `https://account.mekari.com/users/sign_in?app_referer=Talenta` and sign in
+![](img/login.png)
+2. after login on the browser preff `F12`
+3. the new window/new window on bottom screen will popup and switch to `Application` tab, pick `Cookie`->`https://hr.talenta.co` and then scroll down until get the Cookie with name `PHPSESSID` or `_identity`
```
Comment (ans-4175) [inline README.md:23]: I think we need only _identity

File: index.js [hunk 2]
```diff
@@ -1,6 +1,30 @@
 const axios = require('axios');
 const FormData = require('form-data');
 const codec = require('string-codec');
+const readlineSync = require("readline-sync");
```
Comment (ans-4175) [inline index.js]: mending readline ini dipisah saja di file lain, 
Jadi tidak membingungkan index yang cuma ada module talenta-nya buat libs utama di index

Comment (ans-4175) [reply index.js]: index.js: module talenta API
execute.js: app readline data input

File: package.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "talenta-api",
+  "version": "1.0.0",
+  "description": "API HR Talenta for ClockIn and ClockOut",
+  "main": "index.js",
+  "scripts": {
+    "clockin": "node index.js clockin",
```
Comment (ans-4175) [inline package.json]: jadi nanti
`node execute.js clockin`

File: package.json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ans-4175/talenta-api/issues"
+  },
+  "homepage": "https://github.com/ans-4175/talenta-api#readme",
+  "dependencies": {
+    "axios": "^0.22.0",
+    "cheerio": "^1.0.0-rc.10",
```
Comment (ans-4175) [inline package.json]: ga perlu cheerio

**Review by ans-4175 — CHANGES_REQUESTED** [review]
Masih ada beberapa catatan di file index.js, terkait memisahkan module dengan executable app

File: README.md (hunk 1 above)
Comment (k1m0ch1) [reply README.md:23]: I give the option to use the `PHPSESSID` or `_identity` so the user understand that we can use one of it, or what do you think ? just `_identity` is enough?

**Review by ans-4175 — APPROVED** [review]
Still not LGTM

File: README.md (hunk 1 above)
Comment (ans-4175) [reply README.md:23]: hmm, I only use _identity tho, have you tested it for PHPSESSID?
If you have tested it, I think its okay

**Comment by ans-4175** [comment]
For fix #3 I think better to have it separated (execute.js)

**Comment by k1m0ch1** [comment]
> For fix #3 I think better to have it separated (execute.js)

@ans-4175 I'm trying to solve the issue #2 but it is only add `CONTRIBUTING.md` and `PULL_REQUEST_TEMPLATE.md` without change any `README.md`, in order to change `README.md` people need to understand to run the program, get and set the cookie, so I think it is necessary for this PR to need this change, wdyt? 

or is there any less complex explanation to write the `README.md` ?

File: package.json (hunk 4 above)
Comment (k1m0ch1) [reply package.json]: iye, lupa remove pas lagi parsing html ngerjain yg parse cookie

**Comment by ans-4175** [comment]
> people need to understand to run the program, get and set the cookie

I think based on your README, people could understand how to get cookies.
Regarding how to run program, there should be another cases using readLine or this module separately

File: README.md [hunk 5]
```diff
@@ -1,2 +1,58 @@
...
+2. right click on the location
+![](img/get-coor.png)
+3. and select the first menu, your latitude and longitude will be copied and separated with "`,`"
+
+```
+-6.861928521923624, 107.63578698474662
+Latitude          , Longitude
+```
+
+## Run the Program
```
Comment (ans-4175) [inline README.md]: please notes them, this is example.

File: README.md [hunk 6]
```diff
@@ -1,2 +1,58 @@
...
+```
+yarn run clockout
+```
+to clockout
+
+after running that command line, the program will prompt the cookie and coordinate value, just enter if you want to use the default
+
+![](img/program.png)
+
+to remove the input prompt, change the default variable of cookie and coordinate and remove this code
```
Comment (ans-4175) [inline README.md]: this repo (index.js) should only have Talenta API module.
Example of using or any other modules should be separated

File: execute.js [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+let letCookiesTalenta = "_identity=identityCookie"
+cookiesTalenta = "PHPSESSID=asdfasdfasdf"
+let latitude = "-6.861928521923624"
+let longitude = "107.63578698474662"
+
+const dataInput = () => {
+  const inLat = readlineSync.question(`Set Latitude (default: ${latitude}): `)
+  const inLong = readlineSync.question(`Set Longitude (default: ${longitude}): `)
+  const inCookie = readlineSync.question(`Set Latitude (default: ${cookiesTalenta}): `)
```
Comment (ans-4175) [inline execute.js]: should be `Set Cookies`

**Review by ans-4175 — CHANGES_REQUESTED** [review]
just only a small changes

**Comment by k1m0ch1** [comment]
> > people need to understand to run the program, get and set the cookie
> 
> I think based on your README, people could understand how to get cookies. Regarding how to run program, there should be another cases using readLine or this module separately

you right, I will remove the readline for now, and solve this PR for only Issue #2

**Review by ans-4175 — APPROVED** [review]
LGTM

**Pull request merged**